metrics/error-report/Bugsnag.js

```diff
@@ -54,12 +64,19 @@ function initClient(sourceId) {
     appVersion: "process.package_version", // Set SDK version as the app version
     metadata: META_DATA,
     onError: (event) => {
-      const errorOrigin = event.errors[0].stacktrace[0].file;
-      if (typeof errorOrigin === "string") {
-        const index = errorOrigin.lastIndexOf("/");
-        if (!SDK_FILE_NAMES.includes(errorOrigin.substring(index + 1)))
+      const errorOrigin = event.errors[0]?.stacktrace[0];
+      const msg = event.errors[0].errorMessage;
+      if (!errorOrigin || Object.keys(errorOrigin).length === 0) return false;
```

Comment:
Swap these two statements.

Reply:
What is the advantage of doing that?

Reply:
Validations that involve return statements should always be done before any other logic. In this particular case, the impact is not discernable. But the point is to be frugal, "why define the `msg` if there is a chance of discarding it in the immediate statement?".

Reply:
Agreed. Will update.

Reply:
Addressed